assets/base.css

```diff
@@ -2122,33 +2122,47 @@ product-info .loading-overlay:not(.hidden) ~ *,
   }
 }
 
-.utility-bar__grid {
+.utility-bar__grid--3-col,
+.utility-bar__grid--2-col {
   display: grid;
   grid-template-columns: 1fr;
+  grid-template-areas: 'announcements';
 }
 
-.utility-bar__grid .list-social {
+.utility-bar__grid--3-col .list-social,
+.utility-bar__grid--2-col .list-social {
   justify-content: flex-start;
   align-content: center;
   margin-left: -1.2rem;
+  grid-area: social-icons;
 }
 
 @media screen and (max-width: 989px) {
-  .utility-bar .utility-bar__grid .list-social {
+  .utility-bar .utility-bar__grid--3-col .list-social,
+  .utility-bar .utility-bar__grid--2-col .list-social {
     display: none;
   }
+
+  .utility-bar.header__icons--localization {
+    min-height: 0;
+  }
```

Comment:
We're going to need to vertically align the content with `align-items: center;` 👍
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/29-52-g21w9-d6yjs.png)
</details>

We're also going to need to deal with the slider buttons when there are announcements (more than 1). Because the slider from the announcements, has the previous and next button that have a negative margin. Maybe we can make it so that on desktop when there are social icons and language/country picker we cancel that `-3.2rem` margin on the buttons 🤔 

Or we can add a `margin-left: 3.2rem;` when there is more than one block for the locales container 👍 (the same would probably be needed for social icons). 

<details>
<summary>screenshot</summary>

![](https://screenshot.click/29-14-1wwcc-43r6y.png)
</details>